source/iterator/cdc_iterator.go

```diff
@@ -94,6 +94,7 @@ func (w *CDCIterator) Stop() {
 	// stop the two goRoutines
 	w.ticker.Stop()
 	w.tomb.Kill(errors.New("cdc iterator is stopped"))
```

Comment:
It maybe better to log that we are attempting to stop the iterator, and then log that it was stopped after w.tomb.Dead().

Reply:
:+1: